app/src/main/java/nerd/tuxmobil/fahrplan/congress/dataconverters/SessionExtensions.kt

```diff
@@ -164,39 +164,52 @@ fun SessionNetworkModel.toSessionAppModel(): Session {
 }
 
 fun Session.sanitize(): Session {
```

Comment:
Side note: This function could use some documentation explaining why certain modifications are made. Some of the modifications clearly are app-specific and should be part of a specific flavor.

Reply:
I aim at providing reasonable commit messages and unit tests for any change to the `sanitize()` function. Hence, I would **not** like to add dedicated documentation here. It would **duplicate** what can be found in the commit messages and might become **outdated** sooner than later.

![image](https://github.com/EventFahrplan/EventFahrplan/assets/144518/9448c613-5dc7-4dad-88bf-96c1bd7d5d71)

True, there are flavor specific customizations. I left them here because the minor amount is **not worth** splitting now, I think.

Reply:
Forcing people to check the commit history or tests makes it unnecessarily hard to understand the code. In the case of e.g. "Sendezentrum Bühne" neither of those locations gives an explanation for why this was added.

Comments explaining why (not how) something was done rarely have to be changed. Especially not when the code is changed to accomplish the goal in a different way. Such comments usually also give a good indication on when the associated code can be removed because it no longer is relevant.

Reply:
:heavy_check_mark: I see your point. I added documentation now.